runtime/runtime-params-estimator/src/cases.rs

```diff
@@ -110,9 +246,32 @@ pub fn run(config: Config) {
 
     // Measure the speed of adding a function call access key.
     measure_actions(
-        "add access key full",
-        1,
+        Metric::ActionAddFunctionAccessKey1Method,
+        &mut m,
+        &config,
         None,
+        vec![Action::AddKey(AddKeyAction {
+            public_key: serde_json::from_str(
+                "\"ed25519:DcA2MzgpJbrUATQLLceocVckhhAqrkingax4oJ9kZ847\"",
+            )
+            .unwrap(),
+            access_key: AccessKey {
+                nonce: 0,
+                permission: AccessKeyPermission::FunctionCall(FunctionCallPermission {
+                    allowance: Some(100),
+                    receiver_id: get_account_id(0),
+                    method_names: vec!["method1".to_string()],
+                }),
+            },
+        })],
+        true,
+        true,
+    );
+
+    // Measure the speed of adding an access key with 1k methods each 10bytes long.
+    let many_methods: Vec<_> = (0..1000).map(|i| format!("a123456{:03}", i)).collect();
```

Comment:
That's a lot of method names, we should probably limit to X bytes total. The reason is deserializing it every time we need to check a transaction with this access key can be unpaid, but we don't want to waste too much free time. 

It's unrelated to this PR.

Reply:
Acknowledged.

Reply:
https://github.com/nearprotocol/nearcore/issues/1692